tensorboard/components/tf_backend/index.ts

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
In the license, the quotes should be double quotes. There are few rare instances when that is not true and you probably copied from the wrong file :(

Reply:
Done